portal/src/components/interactive-code/LiveProvider.tsx

```diff
@@ -0,0 +1,3 @@
+export {};
```

Comment:
Her også tenker jeg vi kan lage en wrapper rundt `react-live` sin `LiveProvider`. For å gjøre det må vi vel samtidig flytte alt av `import * as buttonReact from "@fremtind/jkl-button-react"` osv.  inn hit.

På den måten er det enklere å legge til ting senere (nye komponenter, styling) hvis vi finner på å ha sånne kodeeditors flere steder.